package.json

```diff
@@ -87,15 +87,14 @@
     },
     "dependencies": {
         "@unibeautify/beautifier-clang-format": "^0.2.0",
-        "@unibeautify/beautifier-csscomb": "^0.1.0",
```

Comment:
Oops looks like a mistake during merge conflict resolution -- deleted `@unibeautify/beautifier-csscomb` 😜 .

Reply:
@Glavin001 isn't it resolved?